app/controllers/api/services_controller.rb

```diff
@@ -36,10 +36,9 @@ def settings
 
   def create
     @service = collection.new # this is done in 2 steps so that the account_id is in place as preffix_key relies on it
-    @service.attributes = service_params
-    @service.backend_api_configs = build_backend_api_configs
+    creator = ServiceCreator.new(service: @service, backend_api: find_backend_api)
```

Comment:
We decided that we should remove the Backend API selector when creating a Service in the UI